docs/getting-started/summaryofchanges.md

```diff
@@ -46,6 +46,11 @@ The following features and enhancements were added.
 #### Zowe installation
 #### API Mediation Layer
 #### Zowe App Server
+
+- Using a cross-memory server without `REUSASID=YES` may result in an ASID shortage. This pull-request adds a check that will print a warning if `REUSASID=YES` is not detected [#145](https://github.com/zowe/zowe-common-c/pull/145)
+- The `InstanceID` parameter within the server predates the environment variable. Therefore, in order to not disturb pre-existing parameters, the server will only use the environment variable if it is non-default. [#130](https://github.com/zowe/zlux-app-server/pull/130)
```

Comment:
That's half of it, the other half:

The App Server will now use the instance.env value of ZOWE_INSTANCE whenever an instance number is needed, such as in the case of determing profile names for RBAC use. Previously, the server instead used the property `InstanceID` instead of `ZOWE_INSTANCE`, but now these are unified when the value of ZOWE_INSTANCE is non-default, in order to maintain backward-compatibility.